sdk-go/wflib/wf_lib_internal.go

```diff
@@ -670,7 +670,9 @@ func (t *ThreadBuilder) handleException(
 	threadName := t.wf.addSubThread(handlerName, handler)
 
 	node.FailureHandlers = append(node.FailureHandlers, &model.FailureHandlerDef{
-		SpecificFailure: exceptionName,
+		FailureToCatch: &model.FailureHandlerDef_SpecificFailure{
+			SpecificFailure: *exceptionName,
```

Comment:
I really don't like the way Go Protobuf works here. The protobuf itself doesn't change, and in Java the code doesn't change to do this. But Go? NOPE